apps/masterbots.ai/components/routes/chat/chat-options.tsx

```
@@ -0,0 +1,128 @@
+} from '@/components/ui/dropdown-menu'
+import { Button } from '@/components/ui/button'
+import {
+  Eye,
+  EyeOff,
+  MoreVertical,
+  Trash,
+  BadgeCheck
+} from 'lucide-react'
+import { useThreadVisibility } from '@/lib/hooks/use-thread-visibility'
+import React from 'react'
+import { Thread } from 'mb-genql'
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)